code/datums/status_effects/debuffs/slime/slimed.dm

```diff
@@ -62,7 +63,7 @@
 /datum/status_effect/slimed/on_apply()
 	if(owner.get_organic_health() <= MIN_HEALTH)
 		return FALSE
-	to_chat(owner, span_userdanger("You have been covered in a thick layer of slime! Find a way to wash it off!"))
+	to_chat(owner, span_userdanger("You have been covered in a thick layer of slime! Find a way to wash it off with water, or try to clean it off by clicking the status effect alert!"))
```

Comment:
You can revert this text now that it glows

Reply:
i still feel its better to communicate this more directly

Reply:
I don't think it's necessary when the glowing effect should convey all that you need to know